docs/reference/dlm/apis/put-lifecycle.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[dlm-put-lifecycle]]
+=== Set the lifecycle of a data stream
+++++
+<titleabbrev>Put Data Stream Lifecycle</titleabbrev>
+++++
+
+experimental::[]
+
+Sets the lifecycle of the targeted data streams.
+
+[[dlm-put-lifecycle-request]]
+==== {api-request-title}
+
+`PUT _data_stream/_lifecycle`
```

Comment:
As above, should we introduce this API ?

Reply:
If we all agree, I would prefer to not introduce it :)